lib/tests/it/main.rs

```diff
@@ -4,7 +4,7 @@ use cap_std::fs::{Dir, DirBuilder};
 use once_cell::sync::Lazy;
 use ostree::cap_std;
 use ostree_ext::chunking::ObjectMetaSized;
-use ostree_ext::container::store::PrepareResult;
+use ostree_ext::container::store;
```

Comment:
It looks like you maybe wanted `ostree_ext::container::store::{self, ImageImporter, PrepareResult}`? But this way is ok too.

Reply:
Hmm...yeah, may consider that in a followup.